.github/actions/setup-idaes/action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
     description: 'Command to use to install `install-target`'
     required: false
     default: pip --no-cache-dir install --progress-bar off
+  ampl-scip-version:
+    description: Version of AMPL Scip solver to install
+    required: false
+    default: '20240121'
```

Comment:
We should not be fixing solver versions wherever possible. In this case, from the issue we are seeing it is likely the failure is because the value being checked can equally be positive or negative (same magnitude) and that the different version of SCIP pick differently. Hopefully, this can be fixed by making the assert check the absolute value of the Pyomo variable instead (both values are equally correct).